src/UpdateLocalizedResources.targets

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="UpdateXlf" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="SaveXlfToResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="UpdateXlfFromResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="EmitLocalizedResources"/>
+
+  <!-- Emit the localized EmbeddedResource items before the common targets start using them -->
+  <PropertyGroup>
+    <PrepareResourceNamesDependsOn>
+      EmitLocalizedResources;
+      UpdateLocalizedResxForEnglishResx;
+      $(PrepareResourceNamesDependsOn)
+    </PrepareResourceNamesDependsOn>
+  </PropertyGroup>
+
+  <Target Name="EmitLocalizedResources">
```

Comment:
> > some parts needed to output items indifferent of incremental check
> I'm not sure I understand. Can you elaborate?

The following target:
```xml
  <Target Name="UpdateLocalizedResxForEnglishResx">

      <ItemGroup>
        <EnglishResources Include="@(NeutralResources -> '$(IntermediateOutputPath)%(Filename).en%(Extension)')"/>
      </ItemGroup>

      <Copy
        SourceFiles="@(NeutralResources)"
        DestinationFiles="@(EnglishResources)"
      />

      <ItemGroup>
        <EmbeddedResource Include="@(EnglishResources)">
          <LogicalName>$(AssemblyName).%(FileName).resources</LogicalName>
        </EmbeddedResource>
      </ItemGroup>      

  </Target>
```

Both produces the intermediary english resx, and adds them to EmbeddedResource. To make copy incremental, I have to extract the item emitting part to another target that always runs after this one. Or, I can move item emitting code in the previous target that emits other items, but then I have to make sure the incremental english resx target runs before that one, so the file specific transforms to compute the logical name work. OK, now that I talked through it it looks feasible :)

> That's definitely interesting. Maybe Touch an additional output if there you're doing test satellites? That would require a clean build to switch back, still.
Aha, make a file lock as an input that gets touched in a previous target if the non-file state is the right one? I see, clever trick :)

Ok then, nailed both problems. Incremental targets it is :)